packages/client/src/proxy.js

```diff
@@ -219,11 +237,31 @@ export function proxyAgentFor(url, options) {
   let { protocol, hostname } = new URL(url);
   let cachekey = `${protocol}//${hostname}`;
 
-  if (!cache.has(cachekey)) {
-    cache.set(cachekey, protocol === 'https:'
-      ? new ProxyHttpsAgent(options)
-      : new ProxyHttpAgent(options));
+  // If we already have a cached agent, return it
+  if (cache.has(cachekey)) {
+    return cache.get(cachekey);
   }
 
-  return cache.get(cachekey);
+  try {
+    let agent;
+    const pacUrl = process.env.PERCY_PAC_FILE_PATH;
```

Comment:
Is this expecting a URL or File path ? or either is fine ? 

considering the most common way would be to use url - we should call this `PERCY_PAC_FILE_URL`

Reply:
It can be either.

> considering the most common way would be to use url - we should call this PERCY_PAC_FILE_URL

Makes sense, naming it `PERCY_PAC_FILE_URL `